front/app/modules/commercial/analytics/admin/components/VisitorsCard/index.tsx

```diff
@@ -35,31 +38,45 @@ const VisitorsCard = ({
     resolution,
   });
 
-  const cardTitle = formatMessage(messages.visitors);
+  const cardTitle = title ? title : formatMessage(messages.visitors);
   const startAt = startAtMoment?.toISOString();
   const endAt = endAtMoment?.toISOString();
 
-  return (
-    <GraphCard
-      title={cardTitle}
-      infoTooltipContent={formatMessage(messages.cardTitleTooltipMessage)}
-      exportMenu={{
+  const exportMenu = interactive
+    ? {
         name: cardTitle,
         svgNode: graphRef,
         xlsx: isNilOrError(xlsxData) ? undefined : { data: xlsxData },
         startAt,
         endAt,
         currentProjectFilter: projectId,
         resolution: currentResolution,
-      }}
+      }
+    : undefined;
+
+  const infoTooltipContent = interactive
+    ? formatMessage(messages.cardTitleTooltipMessage)
+    : undefined;
+
+  const visitorStatBox = interactive ? (
```

Comment:
Similar to what I said about the `TimeControl`: I feel like if we have to add this much conditional logic and even show different statistics in this card, it should maybe be its own component. Then that card could live in the `ReportBuilder` folder, and the `VisitorsStatsReport` could be moved there as well, which makes more sense because that's the only place where it's used.

But anyway, this is related to the module issue, so I think we first have to sort that out